### PR TITLE
More doc on upgrade and config server troubleshooting

### PR DESCRIPTION
**Pull request by hmusum**

### CODE REVIEW

File: en/operations/configuration-server.html [hunk 1]
```diff
@@ -277,14 +277,26 @@ <h2 id="troubleshooting">Troubleshooting</h2>
...
         which will return response code 200 if server is up and has finished bootstrapping.
         Metrics are found at
         <a href="http://myserver0.mydomain.com:12345/state/v1/metrics" data-proofer-ignore>http://myserver0.mydomain.com:12345/state/v1/metrics</a>.
         Use <a href="../reference/vespa-cmdline-tools.html#vespa-model-inspect">vespa-model-inspect</a>
         to find host and port number
       </td>
+    <tr id="consistency">
+      <th>Consistency</th>
+      <td>
+      When having more than one config server consistency between the servers is crucial.
```
Comment (kkraune) [inline en/operations/configuration-server.html:291]:
```suggestion
      When having more than one config server, consistency between the servers is crucial.
```

File: en/operations/configuration-server.html [hunk 2]
```diff
@@ -277,14 +277,26 @@ <h2 id="troubleshooting">Troubleshooting</h2>
...
+      <a href="http://myserver0.mydomain.com:12345/status" data-proofer-ignore>http://myserver0.mydomain.com:12345/status</a>.
```
Comment (kkraune) [inline en/operations/configuration-server.html:292]:
```suggestion
      <a href="http://myserver0.mydomain.com:12345/status" data-proofer-ignore>http://myserver0.mydomain.com:12345/status</a>
```

File: en/operations/configuration-server.html [hunk 3]
```diff
@@ -277,14 +277,26 @@ <h2 id="troubleshooting">Troubleshooting</h2>
...
+      can be used to check that settings for config servers are the same for all servers.
+      <a href="../../en/reference/vespa-cmdline-tools.html#vespa-config-status">vespa-config-status</a> can be used to check config on nodes.
```
Comment (kkraune) [inline en/operations/configuration-server.html:294]:
```suggestion
      <a href="../reference/vespa-cmdline-tools.html#vespa-config-status">vespa-config-status</a> can be used to check config on nodes.
```

File: en/operations/configuration-server.html [hunk 4]
```diff
@@ -277,14 +277,26 @@ <h2 id="troubleshooting">Troubleshooting</h2>
...
+      <a href="http://myserver0.mydomain.com:12345/application/v2/tenant/default/application/default"
+      data-proofer-ignore>http://myserver0.mydomain.com:12345/application/v2/tenant/default/application/default</a>.
+      will active config generation and should be the same on all server and the same as in response from running
```
Comment (kkraune) [inline en/operations/configuration-server.html:297]:
```suggestion
      data-proofer-ignore>http://myserver0.mydomain.com:12345/application/v2/tenant/default/application/default</a>
      displays active config generation and should be the same on all server and the same as in response from running
```

File: en/operations/configuration-server.html [hunk 5]
```diff
@@ -277,14 +277,26 @@ <h2 id="troubleshooting">Troubleshooting</h2>
...
+      <a href="../../en/reference/vespa-cmdline-tools.html#vespa-config-status">vespa-deploy</a>
```
Comment (kkraune) [inline en/operations/configuration-server.html:298]:
```suggestion
      <a href="../reference/vespa-cmdline-tools.html#vespa-config-status">vespa-deploy</a>
```

**Review by kkraune — APPROVED** [review]

**Comment by hmusum** [comment]
Thanks!

**Comment by kkraune** [comment]
👍 merge at will

**Pull request merged**